packages/request/package.json

```diff
@@ -50,25 +51,24 @@
       "injected": true
     }
   },
-  "peerDependencies": {},
   "devDependencies": {
-    "@embroider/addon-dev": "^3.0.0",
-    "rollup": "^3.20.2",
-    "@babel/core": "^7.21.4",
     "@babel/cli": "^7.21.0",
+    "@babel/core": "^7.21.4",
     "@babel/plugin-proposal-class-properties": "^7.18.6",
     "@babel/plugin-proposal-decorators": "^7.21.0",
-    "@babel/plugin-transform-typescript": "^7.21.3",
     "@babel/plugin-proposal-private-methods": "^7.18.6",
     "@babel/plugin-transform-runtime": "^7.21.4",
-    "@babel/preset-typescript": "^7.21.4",
```

Comment:
we can remove 🤨?

Reply:
this is just moving it around since running pnpm install re-ordered the file quite a bit